### ISSUE TITLE
Enable/disable editing functionality needed for collaborative image placement

### ISSUE THREAD

**Issue by ViditChitkara** [publiclab/mapknitter#893]
A follow up of #805 , we'll require an enable/disable button which will have the capability to enable or disable synchronous editing with multiple users on a map. This button should only be shown to the author somewhere on the sidebar:-
<img width="677" alt="Screenshot 2019-07-30 at 11 52 45 PM" src="https://user-images.githubusercontent.com/22616017/62154816-2bb5b000-b325-11e9-96b2-c8b2f185fe0c.png">
We may store a field in the map model like "is_enabled" or something which could be used as a check as to whether changes be accepted from a foreign user or not.

**Comment by ViditChitkara** [comment]
@jywarren @sashadev-sky what do you think?

**Comment by ViditChitkara** [comment]
related issue: #804

**Comment by sashadev-sky** [comment]
Yeah I've been visualizing like a translucent kind of lock icon on the map in a corner let me try to photoshop a pic real quick

**Comment by sashadev-sky** [comment]
The lock idea looked weird :/ The only reasonable thing I came up with is:

![Screen Shot 2019-07-30 at 8 52 06 PM_edited-1](https://user-images.githubusercontent.com/41092741/62176243-b5a64e80-b30e-11e9-897e-ddca35c2e215.jpg)

We could remove everything under "license" and write "you are viewing this map in live editing mode". And the the admin or map owner can click on the green dot to turn off live editing and the box will collapse into something the size of the "write message" area and dot will turn red. The content under "license" will then show up again.

**Comment by ViditChitkara** [comment]
That makes sense. I think we should also remove the upload, post, download etc. Functionality as well--actually everything where a user might be able to edit the map. What do you think?

**Comment by ViditChitkara** [comment]
@jywarren any ideas on this? Should we make it a first-timer-only issue?

**Comment by ViditChitkara** [comment]
Any updates on this?
Thanks

**Comment by jywarren** [comment]
hi!

http://publiclab.org/chat could be a good format for a chat popup or embed. 

What about a "live editing" switch like this? 

![image](https://user-images.githubusercontent.com/24359/63034696-c2d34980-be87-11e9-9fd0-130a11bab0e0.png)

Would it be on by default? Would it be on permanently for anonymous maps?

**Comment by ViditChitkara** [comment]
Seems like a good idea! I was thinking of keeping a field in map table "is_editable" or something. I think it should be on by default. The button idea is great though. I'll make a first timer issue for this.
Thanks

**Comment by jywarren** [comment]
awesome!
